test/functional/fixtures/regression/gh-2067/test.js

```diff
@@ -0,0 +1,17 @@
+describe('[Regression](GH-2067)', function () {
```

Comment:
Could we get more descriptive name for tests and fixture (I mean `describe` section)?